### ISSUE TITLE
how to use webrtc in jetson-utils

### ISSUE THREAD

**Issue by JackeyGuo** [dusty-nv/jetson-utils#141]
Thank you very much for your work. I want to use WebRTC. I hope you can give a tutorial to use WebRTC.

**Comment by dusty-nv** [comment]
@JackeyGuo it's still WIP, but yes there will be tutorial about it

If you have built/installed the `dev` branch of jetson-inference (or `master` of jetson-utils), you can try running something like this:

``` bash
video-viewer /dev/video0 webrtc://@:49567/my_stream
```

Then navigate your browser to `http://$JETSON_IP:49567`

**Comment by JackeyGuo** [comment]
Thanks, but i run the master branch of jetson-utils of the newest version with webrtc output. When i navigate my browser with the seeting port, i can not get the webrtc stream in the browser. The error is like this.
![image](https://user-images.githubusercontent.com/10000640/204753947-e928dd71-fac9-40a4-8c69-ef61050b33b9.png)
![image](https://user-images.githubusercontent.com/10000640/204754637-b365eec5-b249-4eb2-95e3-124a57914b2f.png)

By the way, my env is dgpu with gstreamer 1.20.4.1. I doubt that if it is a version issue.

**Comment by dusty-nv** [comment]
@JackeyGuo have you installed `libgstreamer-plugins-bad1.0-dev` from apt?

I haven't tested this on x86/dGPU, only Jetson platform (aarch64)

**Comment by JackeyGuo** [comment]
Yes, but i install gstreamer from source in dgpu/x86, it must contain this plugin.

**Comment by poett1** [comment]
I tried the same on Xavier NX today with JP 5.0.2, jetson-inference@dev branch & jetson-utils@master branch with the following result:
```
...
[webrtc] HTTP 10.10.0.239 GET '/'
[webrtc] /
[webrtc] websocket /my_stream -- new connection opened by 10.10.0.239 (peer_id=0)
[webrtc] new WebRTC peer connecting (10.10.0.239, peer_id=0)

(video-viewer:59099): GLib-GObject-WARNING **: 11:50:03.008: g_object_set_is_valid_property: object class 'GstWebRTCBin' has no property named 'latency'
**
ERROR:/home/user/Dev/jetson-inference-dev/utils/codec/gstEncoder.cpp:866:static void gstEncoder::onWebsocketMessage(WebRTCPeer*, const char*, size_t, void*): 'sinkpad' should not be nullptr
Bail out! ERROR:/home/user/Dev/jetson-inference-dev/utils/codec/gstEncoder.cpp:866:static void gstEncoder::onWebsocketMessage(WebRTCPeer*, const char*, size_t, void*): 'sinkpad' should not be nullptr
Aborted (core dumped)
```
And Chrome throws `net::ERR_CONNECTION_RESET`. Any hints?
